### PR TITLE
Update npm dependencies

### PR DESCRIPTION
**Pull request by mgechev**
Fixes the dependencies from https://github.com/mgechev/angular2-seed/pull/134

I'm not sure the preinstall tasks are required so I didn't add them, let me know if you still experience this issue without them.

// cc @ravi-tj @jerryorta-dev

### CODE REVIEW

**Comment by jerryorta-dev** [comment]
Just performed a fresh clone of the repo and tested this pr. It works without the preinstall tasks. Simple run ```gulp install``` then ```gulp serve.dev```.

File: package.json [hunk 1]
```diff
@@ -57,7 +57,10 @@
     "tsd": "^0.6.4",
     "typescript": "~1.6.0",
     "zone.js": "^0.5.8",
-    "yargs": "^3.25.0"
+    "yargs": "^3.25.0",
+    "typescript-node": "^0.1.3",
+    "typescript-register": "^1.1.0",
+    "typescript-require": "^0.2.9-1"
```
Comment (tarlepp) [inline package.json]: What about ordering these ASC?

Comment (mgechev) [inline package.json]: Makes sense. Fixed that and removed `typescript-require`, I don't think it is required.

@briantopping would you test against this branch again?

Comment (briantopping) [inline package.json]: Yes, this works flawlessly for me. Pulled changes, removed `node_modules`, `npm install && gulp serve.dev`.

Comment (mgechev) [inline package.json]: Awesome :-)
Thanks!

**Comment by mgechev** [comment]
@ludohenin I think we're ready to merge?

**Comment by briantopping** [comment]
I'd like to see it merged anyway :)

**Comment by ludohenin** [comment]
Done.

**Comment by briantopping** [comment]
Woo hoo, thanks guys! :)

**Comment by Texplode** [comment]
I am getting a tserror: unable to compile typescript 
When I run npm run build.prod

**Pull request merged**